### PR TITLE
[zh-cn] sync pod-lifecycle.md with en page

### PR DESCRIPTION
**Pull request by cnrock**
Add  the EndpointSliceTerminatingCondition section

### CODE REVIEW

**Comment by cnrock** [comment]
sync zh-cn pod-lifecycle.md with en page content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 1]
```diff
@@ -912,6 +912,23 @@ An example flow:
...
+<!--
+{{<note>}}
+If you don't have the `EndpointSliceTerminatingCondition` feature gate enabled
+in your cluster (the gate is on by default from Kubernetes 1.22, and locked to default in 1.26), then the Kubernetes control
+plane removes a Pod from any relevant EndpointSlices as soon as the Pod's
+termination grace period _begins_. The behavior above is described when the
+feature gate `EndpointSliceTerminatingCondition` is enabled.
+{{</note>}}
+-->
+
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:926]:
```suggestion
   {{<note>}}
   <!--
   If you don't have the `EndpointSliceTerminatingCondition` feature gate enabled
   in your cluster (the gate is on by default from Kubernetes 1.22, and locked to default in 1.26), then the    Kubernetes control
   plane removes a Pod from any relevant EndpointSlices as soon as the Pod's
   termination grace period _begins_. The behavior above is described when the
   feature gate `EndpointSliceTerminatingCondition` is enabled.
   -->
   ```

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 2]
```diff
@@ -921,14 +938,14 @@ An example flow:
    to 0 (immediate deletion).
 1. The API server deletes the Pod's API object, which is then no longer visible from any client.
 -->
-4. 超出终止宽限期限时，`kubelet` 会触发强制关闭过程。容器运行时会向 Pod
+1. 超出终止宽限期限时，`kubelet` 会触发强制关闭过程。容器运行时会向 Pod
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:941]:
```suggestion
4. 超出终止宽限期限时，`kubelet` 会触发强制关闭过程。容器运行时会向 Pod
```
不要修改这个编号，原文是对的

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 3]
```diff
@@ -921,14 +938,14 @@ An example flow:
...
    中所有容器内仍在运行的进程发送 `SIGKILL` 信号。
    `kubelet` 也会清理隐藏的 `pause` 容器，如果容器运行时使用了这种容器的话。
 
-5. `kubelet` 触发强制从 API 服务器上删除 Pod 对象的逻辑，并将体面终止限期设置为 0
+1. `kubelet` 触发强制从 API 服务器上删除 Pod 对象的逻辑，并将体面终止限期设置为 0
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:945]:
```suggestion
5. `kubelet` 触发强制从 API 服务器上删除 Pod 对象的逻辑，并将体面终止限期设置为 0
```
不要修改这个编号，原文是对的

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 4]
```diff
@@ -921,14 +938,14 @@ An example flow:
...
    （这意味着马上删除）。
 
-6. API 服务器删除 Pod 的 API 对象，从任何客户端都无法再看到该对象。
+1. API 服务器删除 Pod 的 API 对象，从任何客户端都无法再看到该对象。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:948]:
```suggestion
6. API 服务器删除 Pod 的 API 对象，从任何客户端都无法再看到该对象。
```
不要修改这个编号，原文是对的

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 5]
```diff
@@ -912,6 +912,23 @@ An example flow:
...
+plane removes a Pod from any relevant EndpointSlices as soon as the Pod's
+termination grace period _begins_. The behavior above is described when the
+feature gate `EndpointSliceTerminatingCondition` is enabled.
+{{</note>}}
+-->
+
+{{<note>}} 
+  如果你的集群中没有启用 EndpointSliceTerminatingCondition 功能门
+（该门从 Kubernetes 1.22 开始默认开启，在1.26中锁定为默认），那么一旦 Pod 的终止宽限期开始，Kubernetes 控制平面就会从任何相关EndpointSlices 中移除 Pod。上述行为是在 EndpointSliceTerminatingCondition 功能门被启用时描述的。
+ {{</note>}}
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:930]:
```suggestion
   如果你的集群中没有启用 EndpointSliceTerminatingCondition 特性门控
   （该门控从 Kubernetes 1.22 开始默认开启，在 1.26 中锁定为默认），
   那么一旦 Pod 的终止宽限期开始，Kubernetes 控制平面就会从任何相关 EndpointSlices 中移除 Pod。
  上述行为是在 EndpointSliceTerminatingCondition 特性门控被启用时描述的。
  {{</note>}}
```

**Review by windsonsea — COMMENTED** [review]
Please read through this guide to your localization:

https://kubernetes.io/zh-cn/docs/contribute/localization_zh/

**Comment by windsonsea** [comment]
The [preview](https://deploy-preview-40126--kubernetes-io-main-staging.netlify.app/zh-cn/docs/concepts/workloads/pods/pod-lifecycle/) looks not good now:

image

**Comment by windsonsea** [comment]
aha, it's refreshed and ok now, thanks.

/approve

**Pull request merged**